### PR TITLE
Convert remaining patterns pages to newer Library components

### PR DESCRIPTION
**Pull request by lyzadanger**
This was a quick drop-in conversion from the `<Pattern*>` components to the newer `<Library.*>` components. No significant changes to the demos/contents.

Part of https://github.com/hypothesis/frontend-shared/issues/162

After screenshots, wider viewport. I can provide before screenshots if needed.

image

image

image

This leaves the "Components" pages to convert over to finish out #162

### CODE REVIEW

File: src/pattern-library/components/patterns/FormPatterns.js [hunk 1]
```diff
@@ -1,93 +1,103 @@
...
 import { IconButton, SvgIcon } from '../../../';
 
 export default function FormPatterns() {
   return (
-    <PatternPage title="Forms">
-      <Pattern title="Checkboxes">
+    <Library.Page title="Forms">
+      <Library.Pattern title="Checkbox">
         <p>
           A checkbox, styled accessibly with an SVG image. The SVG image is an
```
Comment (esanzgar) [inline src/pattern-library/components/patterns/FormPatterns.js:10]: Maybe:
```suggestion
          A <code>checkbox</code>, styled accessibly with an SVG image. The SVG image is an
```

File: src/pattern-library/components/patterns/PanelPatterns.js [hunk 2]
```diff
@@ -1,39 +1,46 @@
...
 
 import { IconButton, LabeledButton } from '../../../';
 
 export default function OrganismPatterns() {
   return (
-    <PatternPage title="Panels">
+    <Library.Page title="Panels">
       <p>
         The <code>panel</code> pattern is used by several components:{' '}
         <code>Panel</code>, <code>Dialog</code>, <code>Modal</code>, e.g.
```
Comment (esanzgar) [inline src/pattern-library/components/patterns/PanelPatterns.js:10]: An idea for a future iteration: maybe `<code>Panel</code>, <code>Dialog</code>, <code>Modal</code>` should replaced by links?

File: src/pattern-library/components/patterns/ThumbnailPatterns.js [hunk 3]
```diff
@@ -1,15 +1,10 @@
...
+import Library from '../Library';
 
 import { SvgIcon } from '../../..';
 
 export default function ThumbnailPatterns() {
   return (
-    <PatternPage title="Thumbnails">
+    <Library.Page title="Thumbnails">
       <p>
         The thumbnail pattern is for displaying thumbnail or other images in a
```
Comment (esanzgar) [inline src/pattern-library/components/patterns/ThumbnailPatterns.js:9]:
```suggestion
        The <code>thumbnail</code> pattern is for displaying thumbnail or other images in a
```

**Review by esanzgar — APPROVED** [review]
The `frontend-share` is becoming an amazing resource!

Thank you very much for your hard work on it!

File: src/pattern-library/components/patterns/PanelPatterns.js (hunk 2 above)
Comment (lyzadanger) [reply src/pattern-library/components/patterns/PanelPatterns.js:10]: Indeed: https://github.com/hypothesis/frontend-shared/issues/103 -- already tracked!

**Pull request merged**